dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -885,6 +892,9 @@ serverInitTransactions=Server initiated transactions
 serverInitTransactions.desc=JDBC driver property: SERVER_INITIATED_TRANSACTIONS.
 serverName=Server name
 serverName.desc=Server where the database is running.
+serverNameIgnored.desc=JDBC driver property: serverName. This property is ignored by the driver if a URL is configured.
+serverNameOverrides.desc=JDBC driver property: serverName. This property will override the serverName specified in the URL if one was configured.
+serverNamePostgreSQL.desc=JDBC driver property: serverName. If a URL and this property are configured, then the PostgreSQL driver will ignore this property when using type java.sql.Driver, and ignore the serverName from the URL when using type javax.sql.DataSource.
```

Comment:
Change to:
When the PostgreSQL driver uses type java.sql.Driver and a URL and this property are configured, the PostgreSQL driver ignores this property. When the PostgreSQL driver uses type javax.sql.DataSource, and a URL and this property are configured,  the PostgreSQL driver ignores the serverName property.